app/pages/complete-order/contextCreator.js

```diff
@@ -1,11 +1,16 @@
 import withFundingManifest from './withFundingManifest.json';
 import { baseUrl } from '../../config';
 
+export const addParamsToManifest = (json, params) => JSON.parse(

```

Comment:
can you pop this guy in the helpers, contextCreators folder. There are some unit tests in admin we can also copy over for this function.